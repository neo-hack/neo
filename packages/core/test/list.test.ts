import execa from 'execa'

const cli = require.resolve('../lib/neo.js')

describe('command list', () => {
  it('should contain all templates listed in constants', async () => {
    const { stdout } = await execa('node', [cli].concat(['list']))
    expect(stdout).toMatchSnapshot()
  })
})
