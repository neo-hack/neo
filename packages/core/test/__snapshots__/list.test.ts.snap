// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`command list should contain all templates listed in constants 1`] = `
"
There are 12 templates...

  • rollup-template: rollup stater build pure function lib
  • react-template: webapp starter base on react react-router rematch
  • chrome-extenstion-template: chrome-extenstion starter base ts rxjs react
  • bin-template: bin template
  • react-components-lib-template: react components lib starter, work fine with single component or mutiple components
  • webpack-loader-template: webapck-loader starter
  • ts-lib-template: typescript lib
  • original-html-template: original html without fe framework
  • nextjs-rematch2-template: nextjs and rematch2 quick start
  • eslint-config-template: eslint config template
  • dayjs-plugin-template: dayjs plugin template
  • alfred-workflow-template: alfred workflow


"
`;
